vms/platformvm/add_validator_tx_test.go

```diff
@@ -129,36 +125,13 @@ func TestAddValidatorTxSyntacticVerify(t *testing.T) {
 		t.Fatal("should have errored because rewards owner has no addresses")
 	}
 
-	// Case: Too many shares
```

Comment:
What is the reason for removal?

Reply:
The validator could set a percentage (oddly called Shares) which defined how much of a cut the validator got, when rewards were recieved. This percentage was capped, e.g. a validator could not take 100% of the potential earnings of a delegator. This was a test for that. You can cross reference this with the removal of the check in the SemanticVerify of the addValidatorTx.

Reply:
Ok, understand, even this check is "meaningless" if we start with our fixed stake, it seems for not to be really delegation related and should work in this state as expected.
@evlekht What do you think?

Reply:
IMO this is delegation related, as it tests shares/percentages of the validator which have been removed. The case of testing, if a new validator would take too big of a cut of its delegators is not applicable anymore and hence warrants a removal.

Reply:
Its definitely delegation related and can be removed.
As Jax said, `Shares` its validators cut from delegators earnings. We remove delegating => we remove shares => we can remove this test